src/boot/efi/stdclib.c

```diff
@@ -0,0 +1,74 @@
+/* Public domain */
```

Comment:
I'm sure there's some public domain speedx identifiers...

Reply:
spdx you mean